lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,77 @@
+  const quickRender = rtlRender(NxLoadWrapper, minimalProps);
 
-  it('renders a NxLoadError if there is an error', function() {
-    const ErrorFixture = () => <NxLoadError error="foo" />;
+  describe('when there is an error', ()=> {
+    it('an alert is rendered', ()=> {
+      const componentWithError = renderEl({error: 'Error!'});
+      expect(componentWithError).toBeInTheDocument();
+      expect(componentWithError).toHaveAttribute('role', 'alert');
+      expect(componentWithError?.textContent).toContain('Error!');
+    });
 
-    expect(getShallowComponent({ error: 'foo' })).toMatchElement(<ErrorFixture/>);
-    expect(getShallowComponent({ error: 'foo', loading: true })).toMatchElement(<ErrorFixture />);
-  });
+    it('Retry button is rendered within the alert', ()=> {
+      expect(renderEl()?.textContent).not.toContain('Retry');
+      expect(quickRender({ error: 'Error' }).getByRole('button', { name: 'Retry' }))
+          .toBeInTheDocument();
+    });
 
-  it('passes the retryHandler to NxLoadError', function () {
-    expect(getShallowComponent({ error: 'foo' }).find(NxLoadError)).toHaveProp('retryHandler', retryHandler);
-  });
+    it('retryHandler is called when that Retry button is clicked', async ()=> {
+      const user = userEvent.setup();
+      const retryHandler = jest.fn();
+      expect(renderEl()?.textContent).not.toContain('Retry');
+      expect(quickRender({ error: 'Error', retryHandler }).getByRole('button', { name: 'Retry' }))
+          .toBeInTheDocument();
+      const retryButton = screen.getByRole('button', { name: 'Retry' });
+      expect(retryHandler).not.toHaveBeenCalled();
+      await user.click(retryButton);
+      expect(retryHandler).toHaveBeenCalled();
+    });
+
+    it('children content and loading spinner are not rendered when error is present', ()=> {
+      const toCheck = ['Loading…', 'Foo'];
+      toCheck.forEach(nameToCheck => {
+        expect(renderEl({error: 'Error'})?.textContent).not.toContain(nameToCheck);
+      });
 
-  it('renders a loading spinner if error is not set and loading is true', function() {
-    const SpinnerFixture = () => <NxLoadingSpinner />;
+    });
 
-    expect(getShallowComponent({ loading: true })).toMatchElement(<SpinnerFixture/>);
-    expect(getShallowComponent({ error: 'foo', loading: true })).not.toMatchElement(<SpinnerFixture/>);
   });
 
-  it('renders provided children if loading is false and error is unset', function() {
-    expect(getShallowComponent()).toContainReact(children);
-    expect(getShallowComponent({ loading: false })).toContainReact(children);
-    expect(getShallowComponent({ error: 'foo' })).not.toContainReact(children);
-    expect(getShallowComponent({ loading: true })).not.toContainReact(children);
+  describe('when it is loading', ()=> {
+    expect(renderEl()?.textContent).not.toContain('Loading…');
+    expect(quickRender({ loading: true }).container.textContent).toContain('Loading…');
```

Comment:
There need to be `it` blocks here, you can't just have `expect` calls directly within `describe`.  Additionally, you should test that the children and the error alert are not present.

Reply:
> the children and the error alert are not present.

Done 8c08d918b56a099820d8015327869102c64cf649

Reply:
> > the children and the error alert are not present.
> 
> Done [8c08d91](https://github.com/sonatype/sonatype-react-shared-components/commit/8c08d918b56a099820d8015327869102c64cf649)

Done (modify tests to check role alert) 2f1572ff23dd4c9be90a2fcfee279cdede376a54